test/unittests/libethereum/Transaction.cpp

```diff
@@ -41,19 +41,48 @@ BOOST_AUTO_TEST_CASE(TransactionGasRequired)
 
 BOOST_AUTO_TEST_CASE(TransactionWithEmptyRecepient)
 {
-    // recipient RLP is 0x80 (empty array)
+    // recepient RLP is 0x80 (empty array)
```

Comment:
(Nit) The old spelling is the correct one? https://www.merriam-webster.com/dictionary/recipient

Reply:
oh thanks, my mistake